src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 ## Unreleased
 
+* Added support for gRPC instrumentation under an experimental feature flag
```

Comment:
Re-added/ added back to indicate this is just bringing back an existing feature, but as an experimental, opt-in feature, to reflect the fact that this package is now overall stable, but this specific feature is unstable.